README.md

```diff
@@ -19,6 +19,22 @@ The Charts in the master branch of this repository match the latest packaged Cha
 
 The purpose of this repository is to provide a place for maintaining and contributing official Charts, with CI processes in place for managing the releasing of Charts into the Chart Repository.
 
+The Charts in this repository are organized into two folders:
+* stable
+* incubator
+
+Stable Charts:
+
+1. Provide a method for data persistence
+1. Support application upgrades
+1. Allow customization of the application configuration
+1. Provide a secure default configuration
+1. Do not leverage alpha features of Kubernetes
+
+Incubator Charts are those that do not meet these criteria. Having the incubator folder allows charts to be shared and improved on until they are ready to be moved into the stable folder.
```

Comment:
maybe we should clarify 'those that do not meet these criteria, but still meet the technical and documentation requirements described in CONTRIBUTING.md`. We should also remove https://github.com/kubernetes/charts/pull/42/files/91e80df0945edbfb5a08f545a1a316fecbbc50e7..f3dd89d71b5f5e9a0476ad213ce836a90ad34928#diff-6a3371457528722a734f3c51d9238c13R40 (line mentioning no alpha features)